RecoTracker/Configuration/python/customizeMinPtForHitRecoveryInGluedDet.py

```diff
@@ -13,4 +13,14 @@ def customizeHitRecoveryInGluedDetOn(process):
        process.Chi2MeasurementEstimatorForP5.MinPtForHitRecoveryInGluedDet = 100000
    return process
 
+def customizeHitRecoveryInGluedDetTkSeedsOnly(process):
+   process = customizeMinPtForHitRecoveryInGluedDet(process,0.9)
+   if hasattr(process, "Chi2MeasurementEstimatorForP5"): # keep disabled for cosmics
+       process.Chi2MeasurementEstimatorForP5.MinPtForHitRecoveryInGluedDet = 100000
+   for X in "muonSeededMeasurementEstimatorForOutIn", "hitCollectorForOutInMuonSeeds":
```

Comment:
I did some comparisons and the 80X configuration seems to be a bit better than the 90X configuration of #17807 (gets more hits on good muons, and possibly also slightly less ghosts), so for the legacy I think it would be better to use this 80X version rather than #17807.
I'll make an updated 91X PR to sync it with this one.